wagtail/wagtailadmin/api/endpoints.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import, unicode_literals
+
+from collections import OrderedDict
+
+from wagtail.api.v2.endpoints import DocumentsAPIEndpoint, ImagesAPIEndpoint, PagesAPIEndpoint
+from wagtail.api.v2.filters import (
+    ChildOfFilter, DescendantOfFilter, FieldsFilter, OrderingFilter, SearchFilter)
+from wagtail.api.v2.utils import BadRequestError, filter_page_type, page_models_from_string
```

Comment:
Interesting. I found two definitions of `filter_page_type` and `page_models_from_string`:
* in `wagtail.api.v2.utils`
* in `wagtail.wagtailadmin.views.chooser`

And they are absolutely the same. It's not a subject of this PR, but... Is there any reason to have a copy?